views/guide.lavender

```diff
@@ -322,7 +325,7 @@ block content
             a(
               href='https://www.destroyallsoftware.com/talks/wat', 
               target='_blank') wat
```

Comment:
closing round bracket should always be on an unindented newline if all the attributes aren't on the same line